.gitignore

```diff
@@ -9,4 +9,4 @@ build
 .externalToolBuilders
 .settings
 *flymake.h.gch
-*.user
+CMakeLists.txt.user*
```

Comment:
.gitignore change is unrelated should go into a separate commit.

Reply:
Done